### PR TITLE
samples: ipc_service: Restore default configuration

### PR DESCRIPTION
**Pull request by KAGA164**
This commit allows to build IPC Service sample
for all target in default configuration without
a need for passing additional parameters to
the build command.

### CODE REVIEW

File: samples/ipc/ipc_service/README.rst [hunk 1]
```diff
@@ -66,10 +66,16 @@ Building and running
 **nRF5340 DK**
 
 You can build the sample using either the RPMsg or ICMSG backends, as follows:
+The RPMsg backend is the default one, you can use the following command:
```
Comment (grochu) [inline samples/ipc/ipc_service/README.rst:69]: Just suggesting a bit different wording. Still @peknis has a final word on this.
```suggestion
For the RPMsg backend, that is the default one, you can use the following command:
```

**Review by grochu — COMMENTED** [review]
Just a doc-related comment. The changes look good overall.

File: samples/ipc/ipc_service/README.rst (hunk 1 above)
Comment (peknis) [reply samples/ipc/ipc_service/README.rst:69]: Now, you have two sentences with colon at the end, which is not good. Suggesting the following:

You can build the sample using either the RPMsg or ICMSG backend.
For the default RPMsg backend, use the following command:

File: samples/ipc/ipc_service/README.rst [hunk 2]
```diff
@@ -66,10 +66,16 @@ Building and running
...
+
+.. code-block:: console
+
+   west build -p -b nrf5340dk/nrf5340/cpuapp
+
+For the ICMSG backend, you can use the following command:
```
Comment (peknis) [inline samples/ipc/ipc_service/README.rst:75]:
```suggestion
For the ICMSG backend, use the following command:
```

File: samples/ipc/ipc_service/README.rst [hunk 3]
```diff
@@ -91,17 +97,17 @@ Use these overlays when building the IPC sample to test the following scenarios:
...
-You can build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, as follows:
+You can build the sample to test IPC between the application and radio domains using the :ref:`zephyr:ipc_service_backend_icbmsg` backend that is the default sample configuration, as follows:
 
 .. code-block:: console
 
-   west build -p -b nrf54h20dk/nrf54h20/cpuapp -T sample.ipc.ipc_service.nrf54h20dk_cpuapp_cpuppr_icmsg .
+   west build -p -b nrf54h20dk/nrf54h20/cpuapp
 
-You can build the sample to test IPC between the application and radio domains using the :ref:`zephyr:ipc_service_backend_icbmsg` backend, as follows:
+You can build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, as follows:
```
Comment (peknis) [inline samples/ipc/ipc_service/README.rst:106]:
```suggestion
To build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, use the following command:
```

File: samples/ipc/ipc_service/README.rst [hunk 4]
```diff
@@ -91,17 +97,17 @@ Use these overlays when building the IPC sample to test the following scenarios:
 
 **nRF54H20 DK**
 
-You can build the sample to test IPC between the application and PPR core using the :ref:`zephyr:ipc_service_backend_icmsg` backend, as follows:
+You can build the sample to test IPC between the application and radio domains using the :ref:`zephyr:ipc_service_backend_icbmsg` backend that is the default sample configuration, as follows:
```
Comment (peknis) [inline samples/ipc/ipc_service/README.rst:100]:
```suggestion
To build the sample to test IPC between the application and radio domains using the default :ref:`zephyr:ipc_service_backend_icbmsg` backend, use the following command:
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/ipc/ipc_service/Kconfig.sysbuild [hunk 5]
```diff
@@ -8,3 +8,6 @@ source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
 config REMOTE_BOARD
 	string "The board used for remote target"
+	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP
+	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+	default "nrf54h20dk/nrf54h20/cpurad" if BOARD_NRF54H20DK_NRF54H20_CPUAPP
```
Comment (maje-emb) [inline samples/ipc/ipc_service/Kconfig.sysbuild:13]: Can we also add default configuration for PPR?
Then the sysbuild_cpuppr.conf will be unnecessary.

**Comment by shanthanordic** [comment]
Added DNM to complete testing. Once @miha-nordic  gives +1 it will go in,

File: samples/ipc/ipc_service/Kconfig.sysbuild (hunk 5 above)
Comment (KAGA164) [reply samples/ipc/ipc_service/Kconfig.sysbuild:13]: Not really. Default target for nRF54h20 cpuapp is cpurad

**Review by grochu — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Comment by shanthanordic** [comment]
removing DNM as @miha-nordic  is ooo today but @KAGA164  has tested and confirmed it is working.

**Pull request merged**